### ISSUE TITLE
I'm wondering if this should have been `>=` instead of `>` . The `min_value_and_waste` change policy was previously returning change for excess equal to the min value.

### ISSUE THREAD

**Issue by LLFourn** [bitcoindevkit/coin-select#15]
I'm wondering if this should have been `>=` instead of `>` . The `min_value_and_waste` change policy was previously returning change for excess equal to the min value.

_Originally posted by @jp1ac4 in https://github.com/bitcoindevkit/coin-select/pull/14#discussion_r1439665103_

Create a test for this.

**Comment by jp1ac4** [comment]
For this test, should it also check that if the change policy min value is 0 and the drain value is 0, then `drain_value` still returns `None` (i.e. we don't include a drain with zero value)?

**Comment by LLFourn** [comment]
Well if we keep it as `>` we have to change the name since I read `min` as inclusive.

**Comment by jp1ac4** [comment]
> Well if we keep it as `>` we have to change the name since I read `min` as inclusive.

I agree, I think it's better to change it back to `>=`. I can try working on this if you like.

**Comment by jp1ac4** [comment]
If we use `>=` and the `ChangePolicy` has `min_value` of 0, do you think it should be possible to create a change output with value 0?

**Comment by jp1ac4** [comment]
> If we use `>=` and the `ChangePolicy` has `min_value` of 0, do you think it should be possible to create a change output with value 0?

I think a change ouput with value 0 would be even less likely if the metric controlled the change amount as per #20.
